--- net/minecraft/src/EntitySquid.java
+++ net/minecraft/src/EntitySquid.java
@@ -69,6 +69,7 @@
 	}
 
 	public void onLivingUpdate() {
+        this.kill(); // Goodbye.
 		super.onLivingUpdate();
 		this.field_21062_b = this.field_21063_a;
 		this.field_21059_f = this.field_21061_c;
--- net/minecraft/src/BiomeGenBase.java
+++ net/minecraft/src/BiomeGenBase.java
@@ -40,7 +40,7 @@
 		this.spawnableCreatureList.add(new SpawnListEntry(EntityPig.class, 10));
 		this.spawnableCreatureList.add(new SpawnListEntry(EntityChicken.class, 10));
 		this.spawnableCreatureList.add(new SpawnListEntry(EntityCow.class, 8));
-		this.spawnableWaterCreatureList.add(new SpawnListEntry(EntitySquid.class, 10));
+		//this.spawnableWaterCreatureList.add(new SpawnListEntry(EntitySquid.class, 10));
 	}
 
 	private BiomeGenBase setDisableRain() {
